pkg/convert/testdata/programs/required_providers/main.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_providers {
+    random = {
+      version = "~> 3.5.1"
```

Comment:
Can we translate TF versions to Pulumi versions? I don't think so 1-2-1, but worth asking providers team about it.

Reply:
No we can't really without a bunch of work.  I think we can leave it for now (it is unused and the version string from within the value payload is what is used to parameterize)